cEpiabm/src/sweeps/spatial_sweep.cpp

```diff
@@ -0,0 +1,98 @@
+
+#include "spatial_sweep.hpp"
+#include "../covidsim.hpp"
+#include "../dataclasses/cell.hpp"
+#include "../logfile.hpp"
+
+#include <functional>
+#include <random>
+#include <iterator>
+#include <algorithm>
+
+namespace epiabm
+{
+
+    SpatialSweep::SpatialSweep() {}
+
+    void SpatialSweep::operator()(const unsigned short timestep)
+    {
+        LOG << LOG_LEVEL_DEBUG << "Beginning Spatial Sweep " << timestep;
+        if (m_population->cells().size() <= 1){
+            return;  // no intercell infections if only one cell
```

Comment:
I wonder if clauses like this should be logged - I think that could be quite useful!